provider/cluster/kube/apply.go

```diff
@@ -73,3 +77,33 @@ func applyIngress(kc kubernetes.Interface, b *ingressBuilder) error {
 	}
 	return err
 }
+
+func applyLeaseNS(kc kubernetes.Interface) error {
```

Comment:
`applyManifestNS`?  Maybe we should call our CRD `AkashDeployment`, `Akash` for short.  I dunno.